app-arch/lrzip/patches/lrzip-0.631.patchset

```diff
@@ -0,0 +1,33 @@
+From 5b013fcc5e8a09360bc62101ae73aa4c94b452b9 Mon Sep 17 00:00:00 2001
+From: Leorize <alaviss@users.noreply.github.com>
+Date: Wed, 20 Dec 2017 08:39:56 +0700
+Subject: Move settings folder
+
+
+diff --git a/util.c b/util.c
+index d79125b..b2a055f 100644
+--- a/util.c
++++ b/util.c
+@@ -193,16 +193,16 @@ bool read_config(rzip_control *control)
+ 	if (fp == NULL) {
+ 		HOME=getenv("HOME");
+ 		if (HOME) {
+-			snprintf(homeconf, sizeof(homeconf), "%s/.lrzip/lrzip.conf", HOME);
++			snprintf(homeconf, sizeof(homeconf), "%s/settings/lrzip/lrzip.conf", HOME);
```

Comment:
I guess you meant:
```diff
-snprintf(homeconf, sizeof(homeconf), "%s/settings/lrzip/lrzip.conf", HOME);
+snprintf(homeconf, sizeof(homeconf), "%s/config/settings/lrzip/lrzip.conf", HOME);
```
(your indentation i OK, of course.)
Yep, the system-wide settings dir is /system/settings/, but the user settings dir is /home/config/settings/.

Regarding the ChangeLog file, I haven't read it yet, but I wonder if it would not be better to move it to the devel package.
BTW, I can see that you are using `$developDocDir` instead of `$docDir` + moving it with `packageEntries devel`. I generally choose the second method, but, after all, using `$developDocDir` has its advantages too. I don't know what is the recommendation, if any, on that question.

Reply:
The hardcoded paths to the settings dir should be replaced by calls to find_directory. B_USER_SETTINGS_DIRECTORY should fit. See https://github.com/haikuports/haikuports/wiki/FindDirectory for more info.